### ISSUE TITLE
[Document]  Data of document will not be copied to inherited document

### ISSUE THREAD

**Issue by putzflorian** [pimcore/pimcore#7432]
Data of document will not be copied (in inheritance mode) to target document if source document is only saved and not saved and published once!

**Comment by putzflorian** [comment]
update title (master document wont be applied)

**Comment by putzflorian** [comment]
it seems the master document won't be applied

see the video for a better understanding : https://youtu.be/ngzgpCzX2qI

**Comment by solverat** [comment]
@dvesh3: Just stumbled on that. This is definitely a bug since references via hard link will be updated but an inherited document not.

**Comment by dvesh3** [comment]
@solverat yes, content should be inherited no matter the document is published or not. Somehow, it went unnoticed so we'll look into it. thanks!
